pkg/server/commands/check_command_test.go

```diff
@@ -43,33 +43,10 @@ type doc
 	ts, err := typesystem.NewAndValidate(context.Background(), model)
 	require.NoError(t, err)
 
-	t.Run("validates_store_id", func(t *testing.T) {
-		cmd := NewCheckCommand(mockDatastore, mockCheckResolver, nil)
-		_, _, err := cmd.Execute(context.Background(), &openfgav1.CheckRequest{
-			StoreId: "invalid",
-		})
-		require.ErrorContains(t, err, "invalid CheckRequest.StoreId: value does not match regex pattern \"^[ABCDEFGHJKMNPQRSTVWXYZ0-9]{26}$\"")
-	})
-
-	t.Run("validates_model_id", func(t *testing.T) {
-		cmd := NewCheckCommand(mockDatastore, mockCheckResolver, nil)
-		_, _, err := cmd.Execute(context.Background(), &openfgav1.CheckRequest{
-			StoreId: ulid.Make().String(),
-			TupleKey: &openfgav1.CheckRequestTupleKey{
-				User:     "user:1",
-				Relation: "viewer",
-				Object:   "invalid:1",
-			},
-			AuthorizationModelId: "invalid",
-		})
-		require.ErrorContains(t, err, "invalid CheckRequest.AuthorizationModelId: value does not match regex pattern \"^[ABCDEFGHJKMNPQRSTVWXYZ0-9]{26}$\"")
-	})
-
```

Comment:
these two have been moved to `server_test.go`

Reply:
🤔 why is `server_test.go` a better location for these tests, and for the code itself?

Reply:
validateCheckRequest was doing two related but separate things:

1. grpc contract validation
2. actual check logic validation (object relation user)

This breaks it apart so the [grpc pattern validation](https://github.com/openfga/api/blob/main/openfga/v1/openfga_service.proto#L1107) happens in the server handler before calling the check_command, leaving the check_command responsible only for things related to the logic of check. This makes the check_command entirely separate from the inbound request, which makes sense—if someone sends us an invalid `StoreId` there's no reason to even start a check.

Reply:
I won't block the PR for this, since this is very minor. But i think both validations you mentioned are input validations and I see no reason to put everything inside each individual command.